python/test/testutil.py

```diff
@@ -49,32 +49,29 @@ def setup_for_testing(require_indexes=True, define_queues=[]):
   from google.appengine.api import memcache
   from google.appengine.api import queueinfo
   from google.appengine.datastore import datastore_stub_util
-  from google.appengine.tools import old_dev_appserver
-  from google.appengine.tools import dev_appserver_index
+  from google.appengine.ext import testbed
+  from google.appengine.ext.testbed import TASKQUEUE_SERVICE_NAME
   before_level = logging.getLogger().getEffectiveLevel()
   try:
+    testbed = testbed.Testbed()
     logging.getLogger().setLevel(100)
+    testbed.activate()
```

Comment:
Would have been nicer if that was a Mixin (and maybe using a constant for providing the list of queues to create [if we care do have different ones in different tests). Regardless we should provide a way deactivate testbed and call it it tearDown (explicitly or by mixin).